bin/yaml/libraries.yaml

```diff
@@ -1489,13 +1489,55 @@ libraries:
         targets:
         - trunk
         type: github
+      ztdcuneicode:
+        build_type: cmake
+        make_targets:
+        - ztd.cuneicode
+        check_file: README.md
+        method: nightlyclone
+        repo: soasis/cuneicode
+        targets:
+        - main
+        type: github
```

Comment:
you'll need to help out the builder here a bit by supplying
```
staticliblink:
- ztd.cuneicode
```
and similar to libs below

this will make sure the binaries are properly packaged

Reply:
Ooh, gotcha.

Reply:
Done!